adapters/richaudience/richaudience.go

```diff
@@ -26,36 +26,14 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters
 }
 
 func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
-	raiHeaders := http.Header{}
-
-	setHeaders(&raiHeaders)
-
 	isUrlSecure := getIsUrlSecure(request)
 
-	resImps, err := setImp(request, isUrlSecure)
+	requestDataRequest, err := setImp(request, isUrlSecure, a)
```

Comment:
Is there a reason you are passing the adapter into this function rather than making it an adapter method?

Reply:
I have resolved this in next commit, thanks for your comments